src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -386,6 +405,9 @@ class DeviceAttestationVerifier
                                                            const Crypto::P256PublicKey & dacPublicKey,
                                                            const ByteSpan & csrNonce) = 0;
 
+    virtual AttestationVerificationResult IsCertificateRevoked(bool isPaa, Crypto::AttestationCertVidPid vidPidUnderTest,
```

Comment:
This requires a sync answer.  Probably not OK for this API, which in general might need to do async things (just like verifying the certs above needs to do async things, right?).

Reply:
Oh, I _thought_ this looked familiar.

This _exact_ review feedback was already provided at https://github.com/project-chip/connectedhomeip/pull/30838#issuecomment-1874765776 and that PR was closed and this one was opened. which I can only interpret as a bad-faith way of trying to sneak things past review.  Please do not close PRs and replace them with other PRs that are the same thing.  And _especially_ do not do that if they already have review comments; that's just wasting reviewer time.

Reply:
The intention isn't necessarily to sneak anything in. The old PR was abandoned in favor of a new one with additional changes. 

Your feedback was to make this async in the old PR and it was made async in this PR. Do you still have any concerns?

Reply:
> The intention isn't necessarily to sneak anything in.

Then why were review comments not copied over, making it look like they never happened?

> and it was made async in this PR

Are we looking at the same code?  How does one provide a response from `IsCertificateRevoked` in an async way?

Reply:
IsCertificateRevoked is supposed to be called inside the VerifyAttestationInformation method, which is async. Does this address your concern?

Reply:
The IsCertificateRevoked method also has to be async, since it may require complex access. Overall, a miss of 1.0 APIs is that the AttestationTrustStore execution is not async, and DefaultDACVerifier is not async either, even though, out of all operations, it's one of the longest running ones. Suggest splitting the DeviceAttestation check in the AutoCommissioner to be calling the basics of attestation, prior to doing revocation checks as a second step.

Reply:
> Does this address your concern?

No.